tableau-connector/src/dialect.tdd

```diff
@@ -807,8 +850,8 @@
     <format-date-literal formula="(DATE '%1')" format='yyyy-MM-dd' />
     <format-datetime-literal formula="(TIMESTAMP '%1')" format='yyyy-MM-dd HH:mm:ss.SSS' />
     <format-false literal='FALSE' predicate='FALSE' />
-    <format-is-distinct value='Keyword' />
-    <format-order-by value='Nulls' />
+    <format-is-distinct value='Formula' />
```

Comment:
Changing <format-is-distinct> since we do not support IS NOT DISTINCT FROM syntax. 
Changing <format-order-by> since we do not correctly support NULLS FIRST / LAST syntax.